docs/sphinx/dev_guide/ci_tasks.rst

```diff
@@ -12,99 +12,142 @@
-try updating the submodule to a newer version. If the information
-is still not available, create a branch in the 
-`RADIUSS Spack Configs <https://github.com/LLNL/radiuss-spack-configs>`_ repo, add the needed spec info, and create a pull request.
+A spec containing the compiler and version information for the machine must 
+exist in the `RADIUSS Spack Configs project <https://github.com/LLNL/radiuss-spack-configs>`_. If it does not exist there, create a branch, add the spec entry,
+and create a pull request. After the PR is merged, update the RADIUSS Spack
+Configs submodule in RAJA to be able to run it in GitLab CI.
 
-.. important:: Build spec information used in RAJA Gitlab CI pipelines
+Modifying a configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To change an existing configuration, change the relevant information in the 
+configuration, such variant(s) or target compute architecture, in the 
+appropriate 
+``RAJA/.gitlab/<MACHINE>-build-and-test-extra.yml`` file. Make sure to 
+also modify the job label, so it is descriptive of the configuration.
+
+To modify a shared configuration, it must be changed in the appropriate
+``<MACHINE>-build-and-test.yml`` file in the 
+`RADIUSS Shared CI <https://github.com/LLNL/radiuss-shared-ci>`_ project.
+Create a branch there, modify the job entry, and create a pull request.
+
+.. important:: Build spec information used in RAJA GitLab CI pipelines
                must exist in the ``compilers.yaml`` file and/or 
                ``packages.yaml`` file for the appropriate system type in
                the `RADIUSS Spack Configs <https://github.com/LLNL/radiuss-spack-configs>`_ repo.
 
-Changing Build/Run Parameters
-------------------------------
-
-The commands executed to acquire resources on each 
-system/system-type on which we run Gitlab CI are defined in the 
-`RAJA/.gitlab-ci.yml <https://github.com/LLNL/RAJA/blob/develop/.gitlab-ci.yml>`_ file. The default execution time for each test pipeline is 
-also defined in the file using the variable ``DEFAULT_TIME``. These 
-commands and settings can remain as is for the most part. 
-
-However, sometimes a particular pipeline will take longer to build and
-run than the default allotted time. In this case, the default time can
-be adjusted in the build spec information in the associated 
-``<resource>-jobs.yml`` file discussed in the previous section. 
+               If the desired entry is not there, but exists in a newer version 
+               of the RADIUSS Spack Configs project, update the RAJA submodule 
+               to use the newer version. If the information does not exist in 
+               any version of the RADIUSS Spack Configs project, create a 
+               branch there, add the needed spec info, and create a pull 
+               request. Then, when that PR is merged, update the RAJA submodule
+               to the new version.
```

Comment:
Remove the rest of the repetition.

```suggestion
```

Reply:
I made different changes that should have the same resulting effect.